src/components/calc.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './calc.css';
+// import Button from './Button';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="calculator-display">
+      <input type="text" value="0" readOnly />
+    </div>
```

Comment:
- [ ] As requested earlier, the calculator component should have at least one child component.  You created a `button.js` component but you did not use it. Kindly also  avoid leaving commented-out code on your codebase to keep it clean

Reply:
Fixed.